pkg/ansible/watches/watches.go

```diff
@@ -321,15 +307,28 @@ func Load(path string, maxWorkers, ansibleVerbosity int) ([]Watch, error) {
 		return nil, err
 	}
 
-	watches := []Watch{}
-	err = yaml.Unmarshal(b, &watches)
+	// First we unmarshall to the alias structure
+	alias := []alias{}
+	err = yaml.Unmarshal(b, &alias)
 	if err != nil {
 		log.Error(err, "Failed to unmarshal config")
 		return nil, err
 	}
 
+	// We copy contents from alias structure to the watch structure
+	watches := []Watch{}
+	for _, a := range alias {
+		w, _ := buildWatch(a)
```

Comment:
If we don't care about the error, then why does `buildWatch` return an error? We should never eat errors in non-test code.